cllc-public-app/Models.Extensions/License.cs

```diff
@@ -88,6 +88,20 @@ private static int GetAreaCapacity(string endorsementId, IDynamicsClient dynamic
         }
         private static HoursOfService GetHourService(int dayOfWeek, int? open, int? close)
         {
+            // Occasionally, the function will receive null for open and close, or will recieve the value 845280096. 
```

Comment:
A coding best practice, instead of adding comments here, we can use a constant to make code readable as follow:
const dayClosedValue = 845280096;

Reply:
Updated!

Reply:
@sam-warren  have you pushed your latest changes to github as well? Don't see the new changes.